src/Azure.Functions.Cli/Azure.Functions.Cli.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk" InitialTargets="ExcludeWorkersFromReadyToRun">
+<Project Sdk="Microsoft.NET.Sdk" InitialTargets="ExcludeWorkersFromReadyToRun">
```

Comment:
Assuming the reason there are no changes other than whitespace to this file is because you reverted the WebHost package update? Might as well update this PR to reference your updated preview build version you're testing with for clarity, and as you said, once you move to the actual 4.3.x version with the required changes